spider/tuoitre.py

```diff
@@ -0,0 +1,49 @@
+from functools import partial
+from os.path import basename, splitext
+from urllib.parse import urlparse
+
+from html5lib import parse
+from httpx import ConnectTimeout
+from trio import open_file
+
+INDEX = 'https://tuoitre.vn/vaccine.html'
+
+parse_html5 = partial(parse, namespaceHTMLElements=False)
```

Comment:
Oh, and since this is used by both vnexpress and tuoitre, move it to misc.py and `from .misc import` it.